lib/_emerge/depgraph.py

```diff
@@ -4041,6 +4041,13 @@ def _wrapped_add_pkg_dep_string(
                         inst_pkg, modified_use=self._pkg_use_enabled(inst_pkg)
                     )
                 ]
+                if child and atom.slot_operator == "=":
```

Comment:
Let's add a comment here explaining the invalidation issue.

Reply:
Done. Something like this?